pkg/controller/testing/table.go

```diff
@@ -265,6 +280,23 @@ func (tc *TestCase) VerifyWantAbsent(c client.Client) error {
 	return nil
 }
 
+// VerifyWantEvent verifies that the eventRecorder does contain the events
+// expected to be emitted after reconciliation.
+func (tc *TestCase) VerifyWantEvent(eventRecorder *MockEventRecorder) error {
+	if !reflect.DeepEqual(tc.WantEvent, eventRecorder.events) {
+		return fmt.Errorf("expected %s, got %s", getEventsAsString(tc.WantEvent), getEventsAsString(eventRecorder.events))
+	}
+	return nil
+}
+
+func getEventsAsString(events []corev1.Event) []string {
+	eventsAsString := make([]string, 0)
```

Comment:
`make([]string, 0, len(events))`

Reply:
I read the comment wrong. You are right, fixing it. Thanks!